packages/editor-ui/src/components/TagsDropdown.vue

```diff
@@ -109,7 +118,7 @@ export default mixins(showMessage).extend({
 		}
 
 		if (this.$props.eventBus) {
-			this.$props.eventBus.$on('focus', () => {
+			this.$props.eventBus.on('focus', () => {
```

Comment:
There's a lot of cases where we use `this.$props` in the codebase for no apparent reason, which breaks types suggest. Maybe we could also replace these?

Reply:
Will be tackled as tech debt — [N8N-6113](https://linear.app/n8n/issue/N8N-6313/replace-thisdollarpropspropname-with-thispropname)